skl2onnx/operator_converters/grid_search_cv.py

```diff
@@ -0,0 +1,38 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+from ..common._apply_operation import apply_identity
+from ..common._topology import FloatTensorType
+from ..common._registration import register_converter
+from .._supported_operators import sklearn_classifier_list
+from .._supported_operators import sklearn_operator_name_map
+
+
+def convert_sklearn_grid_search_cv(scope, operator, container):
+    """
+    Converter for scikit-learn's GridSearchCV.
+    """
+    grid_search_op = operator.raw_operator
+    best_estimator = grid_search_op.best_estimator_
+    op_type = sklearn_operator_name_map[type(best_estimator)]
+    grid_search_operator = scope.declare_local_operator(op_type)
+    grid_search_operator.raw_operator = best_estimator
+    grid_search_operator.inputs = operator.inputs
+    label_name = scope.declare_local_variable('label')
+    grid_search_operator.outputs.append(label_name)
+    if type(best_estimator) in sklearn_classifier_list:
+        proba_name = scope.declare_local_variable('probability_tensor',
+                                                  FloatTensorType())
```

Comment:
You need to add a test with the GaussianProcess and the use of double to make sure this code passes.

Reply:
Added GaussianProcessRegressor unit tests with both float and double data types.